app/static/styles/app.css

```diff
@@ -9,7 +9,11 @@
 }
 
 @media -sass-debug-info{filename{}line{font-family:\0000310}}
+<<<<<<< HEAD
```

Comment:
can you reload the page? the css gets generated from scss changes when loading the page